client/src/components/MoviesFiltersContainer/index.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import {Filters, MovieList, SelectedMoviesSection} from "../index";
+import {useMovies} from "../../hooks/useMovies";
+import {useQuery} from "@apollo/client";
+import {MOVIES_QUERY} from "../../pages/Home/queries";
+
+const MoviesFiltersContainer = () => {
+    const {selectedMovies, selectMovie, deleteMovie} = useMovies();
+    const {loading, error, data, refetch} = useQuery(MOVIES_QUERY);
+    if (error) {
+        return 'Error'
+    }
+
+    const onFiltersSubmit =(data) => {
+        console.log(data)
```

Comment:
Why is there a debug message in final version of a task

Reply:
this is not a final version yet

Reply:
Then leave please a todo comment stating what has to be done and in scope of which task (please provide id)